imports/ui/pages/panel/settings/index.js

```diff
@@ -1,18 +1,387 @@
+
+      case "upd-aboutyou":
+        user = {
+          emails: {
+            address:targetValue(target["email2nd"]),
+            verified: false,
+          },
+          profile: {
+            nome: targetValue(target["nome"]),
+            nascimento: targetValue(target["nascimento"]),
+            perfil: targetValue(target["perfil"]),
+            sexo: targetValue(target["sexo"]),
+            especial: $('#especial').is(':checked'),
+          }
+        } 
+
+        Meteor.call('updateUser', user, formID, FlowRouter.getParam('id'), function(error, result) {
+          if (error) {
+            Meteor.call('displayErrors', error);
+          } else if (result) {
+            toastr.success('Dados atualizados!', 'Update!');
+            $('.btn-update').prop('disabled', true);
+          }      
+        });
+
+        break;
+
+      case "upd-address":
+        user = {
+          profile: {
+            endereco: targetValue(target["endereco"]),
+            numero: targetValue(target["numero"]),
+            complemento: targetValue(target["complemento"]),
+            bairro: targetValue(target["bairro"]),
+            cidade: targetValue(target["cidade"]),
+            uf: targetValue(target["uf"]),
+            cep: targetValue(target["cep"])
+          }
+        }
+
+        Meteor.call('updateUser', user, formID, FlowRouter.getParam('id'),function(error, result) {
+          if (error) {
+            Meteor.call('displayErrors', error);
+          } else if (result) {
+            toastr.success('Dados atualizados!', 'Update!');
+            $('.btn-update').prop('disabled', true);
+          }      
+        });
+
+        break;
+
+      case "upd-contact":
+        user = {
+          profile: {
+            phone: targetValue(target["phone"]),
+            celular: targetValue(target["celular"]),
+            facebook: targetValue(target["facebook"]),
+            skype: targetValue(target["skype"]),
+            twitter: targetValue(target["twitter"]),
+            linkedin: targetValue(target["linkedin"]),
+            pers_website: targetValue(target["pers_website"]),
+            github: targetValue(target["github"]),
+          }
+        }
+
+        Meteor.call('updateUser', user, formID, FlowRouter.getParam('id'),function(error, result) {
+          if (error) {
+            Meteor.call('displayErrors', error);
+          } else if (result) {
+            toastr.success('Dados atualizados!', 'Update!');
+            $('.btn-update').prop('disabled', true);
+          }      
+        });
+
+        break;
+
+      case "upd-education":
+        user = {
+          profile: {
+            formacao: {
+              formacao: targetValue(target["formacao"]),
+              curso: targetValue(target["curso"]),
+              conclusao: targetValue(target["conclusao"])
+            }
+          }
+        }
+
+        Meteor.call('updateUser', user, formID, FlowRouter.getParam('id'),function(error, result) {
```

Comment:
You don't need to repeat this same method call over and over, just call it after the switch case